modules/http-api/src/main/java/org/mule/runtime/http/api/client/HttpRequestOptions.java

```diff
@@ -51,6 +51,11 @@ static HttpRequestOptionsBuilder builder(HttpRequestOptions options) {
    */
   Optional<HttpAuthentication> getAuthentication();
 
+  /**
+   * @return if the request should contain a body even for methods without body semantics.
```

Comment:
Please specify which are the HTTP methods without body semantics and a link to the RFC

Reply:
done